src/main/java/com/eerussianguy/firmalife/registry/ModRegistry.java

```diff
@@ -84,6 +96,14 @@
         return allFruitSaps;
     }
 
+    public static ImmutableList<BlockCropDead> getAllDeadCrops() { return allDeadCrops; }
+
+    public static ImmutableList<BlockStemCrop> getAllCropBlocks() { return allCropBlocks; }
+
+    public static BlockStemFruit getPumpkin() { return PUMPKIN_FRUIT; }
```

Comment:
Oh one more thing. I don't see why this is needed if you can just use a lambda or whatever it's called as a supplier. () - > ModRegistry.PUMPKIN_FRUIT

Reply:
Need the supplier at static initializer time, but the fruit isn't non-null until the registry events. So it can't resolve prematurely.

Reply:
apparently I was wrong about this and the lambda does work